### PR TITLE
Update ldbc schema for katana to match the ldbc spec

### PR DESCRIPTION
**Pull request by VhalPurohit**
jira: KAT-5298
As it exists now, the schema used to generate our katana RDGs do not match those in the official spec (link mentioned below). This is because the CSV data generated by the LDBC generator which is used to generate our RDGs contains some additional data (when compared to the official spec), which should ideally be ignored when generating our RDGs. 
This PR edits the schema lists we use to generate our LDBC RDG datasets, so that the data may be regenerated according to the official spec. 
As a side-note, in the future, once we have a way to convert DATETIME to string, we should change the type for all the creationDate(s) in the parquet-string-type datasets to string
This could really use a second pair of eyes, double checking the changes made in the lists and comparing them to the spec here on pg 16 https://ldbcouncil.org/ldbc_snb_docs/ldbc-snb-specification.pdf

### CODE REVIEW

File: csv_datasets/ldbc/lists/nodes-sf-003-parquet-string-type.txt [hunk 1]
```diff
@@ -14,19 +14,19 @@ id:ID(TagClass)|name:STRING|url:STRING
...
+creationDate:DATETIME|id:ID(Comment)|locationIP:STRING|browserUsed:STRING|content:STRING|length:STRING
 sf-003/parquet/32/part.0.parquet
 sf-003/parquet/32/part.1.parquet
 
-creationDate:IGNORE|id:ID(Forum)|title:STRING
+creationDate:DATETIME|id:ID(Forum)|title:STRING
 sf-003/parquet/33/part.0.parquet
 sf-003/parquet/33/part.1.parquet
 
-creationDate:STRING|id:ID(Person)|firstName:STRING|lastName:STRING|gender:STRING|birthday:STRING|locationIP:STRING|browserUsed:STRING
```
Comment (bhargavivadali) [inline csv_datasets/ldbc/lists/nodes-sf-003-parquet-string-type.txt:25]: We are using this file to test type coercion. Can we leave the types unchanged in this file?

Comment (VhalPurohit) [reply csv_datasets/ldbc/lists/nodes-sf-003-parquet-string-type.txt:25]: as of now, there's no available means to cast a datetime to string (it'll throw an error), so we could either ignore the creationDate (and not follow the original schema) or keep the DATETIME. Thoughts @bhargavivadali ?

Comment (bhargavivadali) [reply csv_datasets/ldbc/lists/nodes-sf-003-parquet-string-type.txt:25]: ok

File: csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt [hunk 2]
```diff
@@ -38,7 +38,7 @@ creationDate:IGNORE|:START_ID(Forum)|:END_ID(Post)
 sf-003/parquet/9/part.0.parquet
 sf-003/parquet/9/part.1.parquet
 
-creationDate:IGNORE|:START_ID(Forum)|:END_ID(Person)
+creationDate:DATETIME|:START_ID(Forum)|:END_ID(Person)
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt:41]: so we are not ignoring this any more?  Do you know why we ignored it in the first place?

File: csv_datasets/ldbc/lists/edges-sf-003-half_001.txt [hunk 3]
```diff
@@ -10,40 +10,40 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
 :START_ID(TagClass)|:END_ID(TagClass)|:TYPE
 sf-003/static/tagclass_isSubclassOf_tagclass/part-00000-3a05ed0e-440f-4389-bdd2-7ecabccdedc7-c000.csv.split_1
 
-creationDate:DATETIME|:START_ID(Comment)|:END_ID(Person)|:TYPE
+creationDate:IGNORE|:START_ID(Comment)|:END_ID(Person)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-half_001.txt:13]: I do not follow; why do we ignore in this case?

**Comment by gliga** [comment]
Before I go and look at the details of each file, can you summarize this change.

Can you provide the following:

 * Why the change is being made (explain the motivation)
 * How do the changes different across the data (and why)
 
One big thing to keep in mind is that this might trigger updating schema for all large graphs that are sitting on gs and s3.  Once this is merged, we will need to do any necessary change to large graphs.

File: csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt (hunk 2 above)
Comment (VhalPurohit) [reply csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt:41]: No clue why we were ignoring it initially (@tylershunt might know), but we're not ignoring it anymore since creationDate exists as a property of hasMember (an edge type between Forum and Person)

File: csv_datasets/ldbc/lists/edges-sf-003-half_001.txt (hunk 3 above)
Comment (VhalPurohit) [reply csv_datasets/ldbc/lists/edges-sf-003-half_001.txt:13]: Edge type hasCreator (that goes from node type Comment to node type Person has no creationDate property in the schema). I'm updating the PR description, hopefully that will shed more light on the issue

File: csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt (hunk 2 above)
Comment (gliga) [reply csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt:41]: Can we have comments in this file, so that we can provide more details and links to appropriate places in the spec?

Comment (gliga) [reply csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt:41]: if this is indeed `hasMember` then I confirm this is correct to use DATETIME. 

By the way, did you check that what you have here matches our csv files?

File: csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt [hunk 4]
```diff
@@ -63,12 +63,12 @@ sf-003/parquet/15/part.0.parquet
 sf-003/parquet/15/part.1.parquet
 
 KATANA_DEFAULT_LABEL=KNOWS
-creationDate:IGNORE|:START_ID(Person)|:END_ID(Person)
+creationDate:DATETIME|:START_ID(Person)|:END_ID(Person)
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt:66]: looks good

File: csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt [hunk 5]
```diff
@@ -63,12 +63,12 @@ sf-003/parquet/15/part.0.parquet
...
 sf-003/parquet/16/part.0.parquet
 sf-003/parquet/16/part.1.parquet
 
 KATANA_DEFAULT_LABEL=
-creationDate:IGNORE|:START_ID(Person)|:END_ID(Post)
+creationDate:DATETIME|:START_ID(Person)|:END_ID(Post)
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt:71]: what are edges between Person and Post?  I do not see that in the spec

File: csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt [hunk 6]
```diff
@@ -63,12 +63,12 @@ sf-003/parquet/15/part.0.parquet
 sf-003/parquet/15/part.1.parquet
 
 KATANA_DEFAULT_LABEL=KNOWS
-creationDate:IGNORE|:START_ID(Person)|:END_ID(Person)
+creationDate:DATETIME|:START_ID(Person)|:END_ID(Person)
 sf-003/parquet/16/part.0.parquet
 sf-003/parquet/16/part.1.parquet
 
 KATANA_DEFAULT_LABEL=
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt:70]: why is this empty?

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 7]
```diff
@@ -14,51 +14,51 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
 sf-003/static/tagclass_isSubclassOf_tagclass/part-00000-3a05ed0e-440f-4389-bdd2-7ecabccdedc7-c000.csv.split_0
 sf-003/static/tagclass_isSubclassOf_tagclass/part-00000-3a05ed0e-440f-4389-bdd2-7ecabccdedc7-c000.csv.split_1
 
-creationDate|:START_ID(Comment)|:END_ID(Person)|:TYPE
+:IGNORE|:START_ID(Comment)|:END_ID(Person)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:17]: looks good

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 8]
```diff
@@ -14,51 +14,51 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
...
 sf-003/dynamic/comment_hasCreator_person/part-00000-6fabcea1-6ec8-4944-a25d-c8af3a1810fd-c000.csv.split_0
 sf-003/dynamic/comment_hasCreator_person/part-00000-6fabcea1-6ec8-4944-a25d-c8af3a1810fd-c000.csv.split_1
 
-creationDate|:START_ID(Comment)|:END_ID(Tag)|:TYPE
+:IGNORE|:START_ID(Comment)|:END_ID(Tag)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:21]: looks good

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 9]
```diff
@@ -14,51 +14,51 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
...
 sf-003/dynamic/comment_hasTag_tag/part-00000-17cded5e-b442-4075-97c9-51eb700574b5-c000.csv.split_0
 sf-003/dynamic/comment_hasTag_tag/part-00000-17cded5e-b442-4075-97c9-51eb700574b5-c000.csv.split_1
 
-creationDate|:START_ID(Comment)|:END_ID(Place)|:TYPE
+:IGNORE|:START_ID(Comment)|:END_ID(Place)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:25]: looks good

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 10]
```diff
@@ -14,51 +14,51 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
...
 sf-003/dynamic/comment_isLocatedIn_place/part-00000-b73c3da6-ab3b-4cc1-b5c1-39914cbb9f71-c000.csv.split_0
 sf-003/dynamic/comment_isLocatedIn_place/part-00000-b73c3da6-ab3b-4cc1-b5c1-39914cbb9f71-c000.csv.split_1
 
-creationDate|:START_ID(Comment)|:END_ID(Comment)|:TYPE
+:IGNORE|:START_ID(Comment)|:END_ID(Comment)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:29]: looks good

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 11]
```diff
@@ -14,51 +14,51 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
...
 sf-003/dynamic/comment_replyOf_comment/part-00000-72df4522-f48d-424c-9d01-11bfe0bc7cea-c000.csv.split_0
 sf-003/dynamic/comment_replyOf_comment/part-00000-72df4522-f48d-424c-9d01-11bfe0bc7cea-c000.csv.split_1
 
-creationDate|:START_ID(Comment)|:END_ID(Post)|:TYPE
+:IGNORE|:START_ID(Comment)|:END_ID(Post)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:33]: good

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 12]
```diff
@@ -14,51 +14,51 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
...
 sf-003/dynamic/comment_replyOf_post/part-00000-6bee34cb-0e59-454b-a96f-01bb8c448f42-c000.csv.split_0
 sf-003/dynamic/comment_replyOf_post/part-00000-6bee34cb-0e59-454b-a96f-01bb8c448f42-c000.csv.split_1
 
-creationDate|:START_ID(Forum)|:END_ID(Post)|:TYPE
+:IGNORE|:START_ID(Forum)|:END_ID(Post)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:37]: good

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 13]
```diff
@@ -14,51 +14,51 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
...
 sf-003/dynamic/forum_containerOf_post/part-00000-8e2eab71-bd45-48cb-b856-f62243cd1970-c000.csv.split_0
 sf-003/dynamic/forum_containerOf_post/part-00000-8e2eab71-bd45-48cb-b856-f62243cd1970-c000.csv.split_1
 
 creationDate|:START_ID(Forum)|:END_ID(Person)|:TYPE
 sf-003/dynamic/forum_hasMember_person/part-00000-808cfe18-6222-4e80-8eb7-6805821de22e-c000.csv.split_0
 sf-003/dynamic/forum_hasMember_person/part-00000-808cfe18-6222-4e80-8eb7-6805821de22e-c000.csv.split_1
 
-creationDate|:START_ID(Forum)|:END_ID(Person)|:TYPE
+:IGNORE|:START_ID(Forum)|:END_ID(Person)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:45]: good

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 14]
```diff
@@ -14,51 +14,51 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
...
 sf-003/dynamic/forum_hasModerator_person/part-00000-fb887ce0-a0d5-47c9-b013-ae8aa162462e-c000.csv.split_0
 sf-003/dynamic/forum_hasModerator_person/part-00000-fb887ce0-a0d5-47c9-b013-ae8aa162462e-c000.csv.split_1
 
-creationDate|:START_ID(Forum)|:END_ID(Tag)|:TYPE
+:IGNORE|:START_ID(Forum)|:END_ID(Tag)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:49]: good

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 15]
```diff
@@ -14,51 +14,51 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
...
 sf-003/dynamic/forum_hasTag_tag/part-00000-99e3e635-b013-4902-aad3-03c456b0d52c-c000.csv.split_0
 sf-003/dynamic/forum_hasTag_tag/part-00000-99e3e635-b013-4902-aad3-03c456b0d52c-c000.csv.split_1
 
-creationDate|:START_ID(Person)|:END_ID(Email)|:TYPE
+:IGNORE|:START_ID(Person)|:END_ID(Email)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:53]: what is this one?

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 16]
```diff
@@ -14,51 +14,51 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
...
 sf-003/dynamic/person_email_emailaddress/manually_transformed_2021_03_04.csv.split_0
 sf-003/dynamic/person_email_emailaddress/manually_transformed_2021_03_04.csv.split_1
 
-creationDate|:START_ID(Person)|:END_ID(Tag)|:TYPE
+:IGNORE|:START_ID(Person)|:END_ID(Tag)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:57]: good

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 17]
```diff
@@ -14,51 +14,51 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
...
 sf-003/dynamic/person_hasInterest_tag/part-00000-ac53f591-3322-4242-a3fe-fcadbf18cd73-c000.csv.split_0
 sf-003/dynamic/person_hasInterest_tag/part-00000-ac53f591-3322-4242-a3fe-fcadbf18cd73-c000.csv.split_1
 
-creationDate|:START_ID(Person)|:END_ID(Place)|:TYPE
+:IGNORE|:START_ID(Person)|:END_ID(Place)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:61]: good

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 18]
```diff
@@ -75,28 +75,28 @@ creationDate|:START_ID(Person)|:END_ID(Post)|:TYPE
 sf-003/dynamic/person_likes_post/part-00000-42cce85f-f27a-40d3-8c65-69f139ffd456-c000.csv.split_0
 sf-003/dynamic/person_likes_post/part-00000-42cce85f-f27a-40d3-8c65-69f139ffd456-c000.csv.split_1
 
-creationDate|:START_ID(Person)|:END_ID(Language)|:TYPE
+:IGNORE|:START_ID(Person)|:END_ID(Language)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:78]: where is this coming from?

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 19]
```diff
@@ -75,28 +75,28 @@ creationDate|:START_ID(Person)|:END_ID(Post)|:TYPE
...
 sf-003/dynamic/person_speaks_language/manually_transformed_2021_03_04.csv.split_1
 
-creationDate|:START_ID(Person)|:END_ID(Organisation)|classYear:LONG|:TYPE
+:IGNORE|:START_ID(Person)|:END_ID(Organisation)|classYear:LONG|:TYPE
 sf-003/dynamic/person_studyAt_organisation/part-00000-2156f71d-0672-4640-bb09-0fae5876122e-c000.csv.split_0
 sf-003/dynamic/person_studyAt_organisation/part-00000-2156f71d-0672-4640-bb09-0fae5876122e-c000.csv.split_1
 
 KATANA_DEFAULT_LABEL=WORK_AT
-creationDate|:START_ID(Person)|:END_ID(Organisation)|workFrom:LONG|:IGNORE
+:IGNORE|:START_ID(Person)|:END_ID(Organisation)|workFrom:LONG|:IGNORE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:87]: same question as above good otherwise

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 20]
```diff
@@ -75,28 +75,28 @@ creationDate|:START_ID(Person)|:END_ID(Post)|:TYPE
...
 sf-003/dynamic/person_workAt_organisation/part-00000-274143b8-ede7-4a41-b8c8-10fea2aa3056-c000.csv.split_0
 sf-003/dynamic/person_workAt_organisation/part-00000-274143b8-ede7-4a41-b8c8-10fea2aa3056-c000.csv.split_1
 
 KATANA_DEFAULT_LABEL=
-creationDate|:START_ID(Post)|:END_ID(Person)|:TYPE
+:IGNORE|:START_ID(Post)|:END_ID(Person)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:92]: good

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 21]
```diff
@@ -75,28 +75,28 @@ creationDate|:START_ID(Person)|:END_ID(Post)|:TYPE
...
 sf-003/dynamic/post_hasCreator_person/part-00000-10cb9ef3-dc6c-43e4-b6e7-a2179dcd810c-c000.csv.split_0
 sf-003/dynamic/post_hasCreator_person/part-00000-10cb9ef3-dc6c-43e4-b6e7-a2179dcd810c-c000.csv.split_1
 
-creationDate|:START_ID(Post)|:END_ID(Tag)|:TYPE
+:IGNORE|:START_ID(Post)|:END_ID(Tag)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:96]: good

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 22]
```diff
@@ -75,28 +75,28 @@ creationDate|:START_ID(Person)|:END_ID(Post)|:TYPE
...
 sf-003/dynamic/post_hasTag_tag/part-00000-8f918eb1-beed-428f-b68a-18f1c5e3d1d0-c000.csv.split_0
 sf-003/dynamic/post_hasTag_tag/part-00000-8f918eb1-beed-428f-b68a-18f1c5e3d1d0-c000.csv.split_1
 
-creationDate|:START_ID(Post)|:END_ID(Place)|:TYPE
+:IGNORE|:START_ID(Post)|:END_ID(Place)|:TYPE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:100]: good

File: csv_datasets/ldbc/lists/edges-sf-003-full.txt [hunk 23]
```diff
@@ -75,28 +75,28 @@ creationDate:DATETIME|:START_ID(Person)|:END_ID(Post)|:TYPE
...
 sf-003/dynamic/post_hasCreator_person/part-00000-10cb9ef3-dc6c-43e4-b6e7-a2179dcd810c-c000.csv.split_1
 
-creationDate:DATETIME|:START_ID(Post)|:END_ID(Tag)|:TYPE
+creationDate:IGNORE|:START_ID(Post)|:END_ID(Tag)|:TYPE
 sf-003/dynamic/post_hasTag_tag/part-00000-8f918eb1-beed-428f-b68a-18f1c5e3d1d0-c000.csv.split_0
 sf-003/dynamic/post_hasTag_tag/part-00000-8f918eb1-beed-428f-b68a-18f1c5e3d1d0-c000.csv.split_1
 
-creationDate:DATETIME|:START_ID(Post)|:END_ID(Place)|:TYPE
+creationDate:IGNORE|:START_ID(Post)|:END_ID(Place)|:TYPE
 sf-003/dynamic/post_isLocatedIn_place/part-00000-d2a7a9ca-9711-48ec-b36d-3089c9a62f09-c000.csv.split_0
```
Comment (gliga) [inline csv_datasets/ldbc/lists/edges-sf-003-full.txt:101]: this file looks good; same comments as in the previous one obviously, but matching

File: csv_datasets/ldbc/lists/edges-sf-003-half_001.txt (hunk 3 above)
Comment (gliga) [reply csv_datasets/ldbc/lists/edges-sf-003-half_001.txt:13]: The description now is very good.

File: csv_datasets/ldbc/lists/nodes-sf-003-parquet-string-type.txt [hunk 24]
```diff
@@ -14,19 +14,19 @@ id:ID(TagClass)|name:STRING|url:STRING
...
+creationDate:DATETIME|id:ID(Person)|firstName:STRING|lastName:STRING|gender:STRING|birthday:STRING|locationIP:STRING|browserUsed:STRING
 sf-003/parquet/34/part.0.parquet
 sf-003/parquet/34/part.1.parquet
 
-creationDate:IGNORE|id:ID(Post)|imageFile:STRING|locationIP:STRING|browserUsed:STRING|language:STRING|content:STRING|length:STRING|forumID:STRING
+creationDate:DATETIME|id:ID(Post)|imageFile:STRING|locationIP:STRING|browserUsed:STRING|language:STRING|content:STRING|length:STRING|forumID:IGNORE
```
Comment (gliga) [inline csv_datasets/ldbc/lists/nodes-sf-003-parquet-string-type.txt:29]: based on the parts it is hard to know what is in the files; we should have a comments or labels what nodes are being here described.

**Review by gliga — APPROVED** [review]
Good overall.

Check some questions/comments.

**Comment by gliga** [comment]
Super important: Please do NOT close this PR before creating a Jira ticket that header files on gs and s3 have to be updated.

File: csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt (hunk 5 above)
Comment (roshandathathri) [reply csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt:71]: Every Post is also a Message (it's a derived type). This edge is to indicate that a person "likes" a post.

File: csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt (hunk 6 above)
Comment (roshandathathri) [reply csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt:70]: Don't know but it doesn't matter if every file has a specific label. There should be no default/empty labels for LDBC.

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt (hunk 18 above)
Comment (roshandathathri) [reply csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:78]: Second Milos. I don't see a "Language" node in the spec.

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 25]
```diff
@@ -75,28 +75,28 @@ creationDate|:START_ID(Person)|:END_ID(Post)|:TYPE
 sf-003/dynamic/person_likes_post/part-00000-42cce85f-f27a-40d3-8c65-69f139ffd456-c000.csv.split_0
 sf-003/dynamic/person_likes_post/part-00000-42cce85f-f27a-40d3-8c65-69f139ffd456-c000.csv.split_1
 
-creationDate|:START_ID(Person)|:END_ID(Language)|:TYPE
+:IGNORE|:START_ID(Person)|:END_ID(Language)|:TYPE
 sf-003/dynamic/person_speaks_language/manually_transformed_2021_03_04.csv.split_0
 sf-003/dynamic/person_speaks_language/manually_transformed_2021_03_04.csv.split_1
 
-creationDate|:START_ID(Person)|:END_ID(Organisation)|classYear:LONG|:TYPE
+:IGNORE|:START_ID(Person)|:END_ID(Organisation)|classYear:LONG|:TYPE
```
[the remark this answers is not in the document]
Comment (roshandathathri) [reply csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:82]: The only integer type in Cypher is 64-bit signed integer but I'm with Milos here that we should be using 32-bit integer as the spec says it's 32 bits.

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt (hunk 19 above)
Comment (roshandathathri) [reply csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:87]: Ditto

**Review by roshandathathri — APPROVED** [review]
LGTM too. Thanks everyone!

**Comment by roshandathathri** [comment]
> Super important: Please do NOT close this PR before creating a Jira ticket that header files on gs and s3 have to be updated.

Agree. This will also need to be socialized/announced. We'll need to update graphs in test-datasets and in gs/s3. People need to know that the data in the graph has changed, not just the format.

File: csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt (hunk 6 above)
Comment (VhalPurohit) [reply csv_datasets/ldbc/lists/edges-sf-003-full-parquet.txt:70]: @gliga How do you mean empty?

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt [hunk 26]
```diff
@@ -75,28 +75,28 @@ creationDate|:START_ID(Person)|:END_ID(Post)|:TYPE
...
 sf-003/dynamic/person_studyAt_organisation/part-00000-2156f71d-0672-4640-bb09-0fae5876122e-c000.csv.split_0
 sf-003/dynamic/person_studyAt_organisation/part-00000-2156f71d-0672-4640-bb09-0fae5876122e-c000.csv.split_1
 
 KATANA_DEFAULT_LABEL=WORK_AT
-creationDate|:START_ID(Person)|:END_ID(Organisation)|workFrom:LONG|:IGNORE
+:IGNORE|:START_ID(Person)|:END_ID(Organisation)|workFrom:LONG|:IGNORE
 sf-003/dynamic/person_workAt_organisation/part-00000-274143b8-ede7-4a41-b8c8-10fea2aa3056-c000.csv.split_0
 sf-003/dynamic/person_workAt_organisation/part-00000-274143b8-ede7-4a41-b8c8-10fea2aa3056-c000.csv.split_1
 
 KATANA_DEFAULT_LABEL=
```
[the remark this answers is not in the document]
Comment (VhalPurohit) [reply csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:91]: @tylershunt might have some context

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt (hunk 25 above)
[the remark this answers is not in the document]
Comment (roshandathathri) [reply csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:82]: Properties can be other integer types. We need to handle type-casting in the engine, which we do already.

**Review by e-mcginnis — CHANGES_REQUESTED** [review]
> Agree. This will also need to be socialized/announced. We'll need to update graphs in test-datasets and in gs/s3. People need to know that the data in the graph has changed, not just the format.

@VhalPurohit 

Sorry for the hard block, but If this is going to lead to changes in the graph, I would really like to see the rdgs `rdg_datasets/ldbc_003*` updated also as part of this PR, as well as a CI run with the new RDGs so we know what tests will need to be updated. 

Feel free to reach out if you need any help with this.

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt (hunk 15 above)
Comment (VhalPurohit) [reply csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:53]: @gliga this looks like one of those edges that don't exist in the actual schema but came out of the csv generator for some reason (idk why the generator would generate something not in the schema, @tylershunt / @danielmawhirter  might have more context)

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt (hunk 18 above)
Comment (VhalPurohit) [reply csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:78]: @gliga @roshandathathri  this looks like one of those edges that don't exist in the actual schema but came out of the csv generator for some reason (idk why the generator would generate something not in the schema, @tylershunt / @danielmawhirter might have more context)

File: csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt (hunk 19 above)
Comment (VhalPurohit) [reply csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt:87]: replied above

File: csv_datasets/ldbc/lists/nodes-sf-003-parquet-string-type.txt (hunk 24 above)
Comment (VhalPurohit) [reply csv_datasets/ldbc/lists/nodes-sf-003-parquet-string-type.txt:29]: @danielmawhirter any idea how we could map each of these parquet files to the corresponding node/ edge it belongs to?

**Review by scober — CHANGES_REQUESTED** [review]
Placing my own lock so that @e-mcginnis can remove his as part of the transition

**Comment by VhalPurohit** [comment]
Closing because this was handles by a bunch of other PRs, the last of which was Github: #31
